File: Python/Function/05.evennumsum.py
#Write a Function to add all the even number from 1 to N

N = int(input("Enter any number: "))

def totalSum(num):
    sum = 0
    
    for i in range(2, num, 2):
        sum += i
    
    return sum
        
result = totalSum(N)

print("Sum of all the even numbers is: ", result)

File: CPP/Function/05.evennumsum.cpp
//Write a Function to add all the even number from 1 to N

#include <iostream>

using namespace std;

int totalSum(int num){
    int sum = 0;
    
    for (int i =2; i < num; i=i+2){
        sum = sum+i;
    }
    
    return sum;
}

int main()
{
    
    int a;
    cout<<"Enter the number: ";
    cin>>a;
    int result = totalSum(a);
    cout<<"Sum of all the even number is: "<< result ;
    
    return 0;
}